### ISSUE TITLE
group deactivation

### ISSUE THREAD

**Issue by kmahar** [haystack/murmur#64]
It's not clear what it means to deactivate a group, and reactivating is difficult. I have just deactivated one, and I am still able to send posts to it, both from the Murmur interface and via email. The only real change seems to be that it doesn't show up on the "My Groups" tab. But it still shows up under the dropdown list of groups next to the Murmur logo. It's very difficult to get back to the group info page to reactivate it -- have to go the group on the dropdown list, then "change your settings", then finally get a link to the group info page.

**Comment by amyxzhang** [comment]
Yeah, the activate variable was created at some point and some initial buttons were added but we never actually did *anything* with it.
